images/tier3/notstarted-circle.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="30" height="30" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Make sure you have a `viewBox="0 0 30 30"` on the `svg` element -- that's for IE.